model/h2oautoml/dffml_model_h2oautoml/h2oautoml.py

```diff
@@ -0,0 +1,336 @@
+    )
+    keep_cross_validation_models: bool = field(
+        "keep models of cross-validation", default=False
+    )
+    keep_cross_validation_fold_assignment: bool = field(
+        "Preserve the cross-validation fold assignment", default=False
+    )
+    verbosity: str = field("Print the backend messages", default=None)
+    export_checkpoints_dir: str = field(
+        "Specify the directory to save the models", default=None
+    )
+    show_leaderboard: bool = field(
+        "Print the leaderboard after the building the models in AutoML",
+        default=False,
+    )
+
+
+@entrypoint("h2oautoml")
+class H2oAutoMLModel(SimpleModel):
+    r"""
+    H2O’s AutoML can be used for automating the machine learning workflow,
+    which includes automatic training and tuning of many models within a user-specified time-limit
+
+    Examples
+    --------
+
+    Command line usage
+
+    First download the training and test files, change the headers to DFFML
+    format.
+
+    .. code-block::
+        :test:
+
+        $ wget http://download.tensorflow.org/data/iris_training.csv
+        $ wget http://download.tensorflow.org/data/iris_test.csv
+        $ sed -i 's/.*setosa,versicolor,virginica/SepalLength,SepalWidth,PetalLength,PetalWidth,classification/g' iris_training.csv iris_test.csv
+
+
+    Run the train command
+
+    .. code-block:: console
+        :test:
+
+        $ dffml train \
+            -sources train=csv \
+            -source-filename iris_training.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model \
+            -model-max_models 5
+
+    Assess the accuracy
+
+    .. code-block:: console
+        :test:
+
+        $ dffml accuracy \
+            -sources train=csv \
+            -source-filename iris_test.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model
+            
+    Output
+
+    .. code-block::
+
+        accuracy: 0.8685475825
+
+
+    Make predictions
+
+    .. code-block:: console
+        :test:
+
+        $ dffml predict all \
+            -sources train=csv \
+            -source-filename iris_test.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model
+
+    Python usage
+
+    **run.py**
+
+    .. literalinclude:: ../model/h2oautoml/examples/example_h2oautoml.py
```

Comment:
This path is being included from `docs/plugins/dffml_model.rst`

We need to prefix with a `/` to make sure that the path is relative to the root of the docs.

```suggestion
    .. literalinclude:: /../model/h2oautoml/examples/example_h2oautoml.py
```